rpcserver.go

```diff
@@ -3948,8 +3944,7 @@ func (s *rpcServer) jsonRPCRead(w http.ResponseWriter, r *http.Request, isAdmin
 	if err != nil {
 		errMsg := fmt.Sprintf("error reading JSON message: %v", err)
 		errCode := http.StatusBadRequest
-		http.Error(w, strconv.FormatInt(int64(errCode), 10)+" "+errMsg,
-			errCode)
+		http.Error(w, strconv.Itoa(errCode)+" "+errMsg, errCode)
```

Comment:
Um actually after looking at the full context, it's already using fmt.Sprintf.  Can we do it all at once there instead?